src/test/clojure/jarl/builtins/aggregates_test.clj

```diff
@@ -1,64 +1,55 @@
 (ns jarl.builtins.aggregates_test
-  (:require [clojure.test :refer [deftest is testing]]
-            [jarl.builtins.aggregates :refer [builtin-count builtin-sum builtin-product
-                                              builtin-max builtin-min builtin-sort]])
-  (:import (se.fylling.jarl BuiltinException)))
+  (:require [clojure.test :refer [deftest]]
+            [test.utils :refer [testing-builtin]])
+  (:import (se.fylling.jarl TypeException)))
 
 (deftest builtin-count-test
-  (testing "count"
-    (is (= (builtin-count []) 0))
-    (is (= (builtin-count [1]) 1))
-    (is (= (builtin-count [1 "a"]) 2))
-    (is (= (builtin-count "abc") 3))
-    (is (= (builtin-count "🍧🍨🧁🍰🍮") 5)))
-  (testing "multiple codepoints"
-    (is (= (builtin-count "🇩🇪") 2))))
+  (testing-builtin "count"
+    [[]]              0
+    [[1]]             1
+    [[1 "a"]]         2
+    ["abc"]           3
+    [(range 11)]      11
+    ["🍧🍨🧁🍰🍮"]   5
```

Comment:
That misalignment is a bit of an eyesore 😄 .

Reply:
Yes, but emojis aren't fixed length I'm afraid :(